### PR TITLE
No Bug: Fixed BundleId for Beta BraveShareTo for submissions.

### PR DESCRIPTION
**Pull request by jhreis**
## Summary of Changes

This pull request fixes issue #<number>

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/internal/issues/new?assignees=&labels=security%2C+security-reviews&template=security-review.md&title=iOS:%20Security+Review+for) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by kylehickinson — APPROVED** [review]

**Comment by jhreis** [comment]
@iccub is not having any issues with this. So going to block this for now.

@iccub you are free to unblock this if you decide this is useful / needed (e.g. you or someone else cannot submit betas)

**Comment by iccub** [comment]
Will reopen if we still have issues with it

**Pull request closed without merging**
